benchmark-runner/analyze.ts

```diff
@@ -26,8 +27,31 @@ export interface HyperfineResult {
   exit_codes: number[];
 }
 
+function resolveVersion(pkgName: string): {pkgName: string, version: string} {
+  return  {
+    pkgName,
+    version: execSync(`npm view ${pkgName} version`).toString('utf-8').split('\n').join('')
```

Comment:
> [By default, the npm view command returns the latest version of the package](https://www.javascripttutorial.net/nodejs-tutorial/npm-view/)

I think you should get the version of the locally installed package. It seems `npm view` doesn't do that.

Reply:
Most time, we run benchmarks in CI, and the `@speedy-js/speedy-core` is [linked](https://github.com/hardfist/speedystack/blob/17641b31ef3e006ff2d24f56ec7da25fbab9a8a4/.github/workflows/Benchmark.yml#L39-L40) to the folder rather than installed.